python/main.py

```diff
@@ -23,10 +26,38 @@ def root():
     return {"message": "Hello, world!"}
 
 @app.post("/items")
-def add_item(name: str = Form(...)):
+def add_item(name: str = Form(...), category: str = Form(...)):
     logger.info(f"Receive item: {name}")
+
+    # json format to be written
+    items_list = {"items" : []}
+
+    # if item.json exsits, load the data first
+    if os.path.isfile('items.json') == True:
```

Comment:
nit: you can omit the equality condition as follows
```suggestion
    if os.path.isfile('items.json'):
```

Reply:
Thank you. Changed as suggested.